content/en/about/faq/distributed-tracing/index.md

```diff
@@ -4,6 +4,7 @@ linktitle: Distributed Tracing
 description: Distributed Tracing Q & A.
 weight: 46
 layout: faq
+icon: faq
```

Comment:
Page icons have been removed; this change isn't necessary.  Please remove it from the pull request.

Reply:
resolved